### PR TITLE
Add configuration options

### LINKED ISSUE #2: Add the ability to tweak or disable rules
We use different rules. An easy way tweak or disable existing ones would be great.

### PR DESCRIPTION
**Pull request by supremebeing7**
Fixes #2 

There's one piece that I have not done yet because I wanted to get your input, @mmozuras:
renaming the `FiftyCharSummary` and `SeventyTwoCharLimit` classes. Since those are now configurable, it seems odd to have a specific number as a part of the class name. Would you agree?

Also, if you like I can squash the commits down to one.

Let me know what you think.

### CODE REVIEW

**Comment by mmozuras** [comment]
> Since those are now configurable, it seems odd to have a specific number as a part of the class name. Would you agree?

@supremebeing7 completely agree!

File: lib/poper/rule/fifty_char_summary.rb [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+        @config.summary_character_limit_enabled.to_s == 'true'
+      end
+
+      private
+
+      def character_limit
+        @config.summary_character_limit_number.to_i
+      end
+
+      def error_message
```
Comment (mmozuras) [inline lib/poper/rule/fifty_char_summary.rb]: @supremebeing7 why did you change it from a constant to a method?

Comment (supremebeing7) [inline lib/poper/rule/fifty_char_summary.rb]: I had to do that for the character count rules in order to interpolate the number of characters. The rest I changed for consistency.

Comment (mmozuras) [inline lib/poper/rule/fifty_char_summary.rb]: @supremebeing7 👌 

Do [these](https://github.com/mmozuras/poper/pull/6#issuecomment-291006940) and I'm ready to merge! 😄

Comment (supremebeing7) [inline lib/poper/rule/fifty_char_summary.rb]: Done.

**Comment by mmozuras** [comment]
@supremebeing7 perfect 👍

**Comment by mmozuras** [comment]
Released as [v0.2.0](https://github.com/mmozuras/poper/releases/tag/v0.2.0)!
